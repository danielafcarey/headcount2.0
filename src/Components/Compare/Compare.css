.compare-data {
  background: url('../../Images/compareImage.jpg');
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  background-size: cover;
  background-repeat: no-repeat;
  min-width: 40%;
  min-height: 100%;
  width: auto;
  height: auto;
}

.compareDataDesign {
  text-align: right;
  padding-right: 42px;
  padding-top: 20px;
  margin: 0;
}

.compare{
  text-align: right;
  padding-right: 139px;
  padding-top: 20px;
  margin: 0;
}

.compareSpace {
  padding-top: 63px;
}

.font {
  font-family: 'Press Start 2P';
  font-size: small;
  text-shadow: 1px 1px white;
}
