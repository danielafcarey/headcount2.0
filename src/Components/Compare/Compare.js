import React from 'react';
// import Compare from '../Compare/Compare.js';
import PropTypes from 'prop-types';

const Compare = ({ compareCards, compareData }) => {

  return (
    <div>
     
    </div>
    )
}

Compare.propTypes = {
  compareCards: PropTypes.array.isRequired
}

export default Compare;