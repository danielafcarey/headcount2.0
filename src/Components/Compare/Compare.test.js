import React from 'react';
import { shallow, mount } from 'enzyme';
import Compare from './Compare.js';
import PropTypes from 'prop-types';
import renderer from 'react-test-renderer';

describe('Compare', () => {
  let compare;

  beforeEach(() => {
    compare = shallow(<Compare compareCards={[]} 
                     compareData={{}}
                     removeCompareCard={jest.fn()}
            />);
  })

  it('matches the snapshot with no cards', () => {
    const compareSnapshot = renderer.create(<Compare compareCards={[]} 
                                                     compareData={{}}
                                                     removeCompareCard={jest.fn()} 
                            />).JSON;
    expect(compareSnapshot).toMatchSnapshot();
  })

  it('matches the snapshot when one card is in compareCards prop', () => {
    const mockCompareCards = [
      { 'COLORADO': [{'2009': 1}], 
        selected: false 
      }
    ];
    const compareSnapshot = renderer.create(<Compare compareCards={ mockCompareCards } 
                                                     compareData={{}}
                                                     removeCompareCard={jest.fn()} 
                            />).JSON;
    expect(compareSnapshot).toMatchSnapshot();

  })

  it('matches the snapshot when two cards in compareCards prop', () => {
    const mockCompareCards = [
      { 'COLORADO': [{'2009': 1}], 
        selected: true 
      },
      { 'AGATE 300': [{'2010': 0}], 
        selected: true 
      }
    ];
    const mockCompareData = {
      'COLORADO': 1,
      'AGATE 300': 0,
      compared: 1
    };
    const compareSnapshot = renderer.create(<Compare compareCards={ mockCompareCards } 
                                                     compareData={ mockCompareData }
                                                     removeCompareCard={jest.fn()} 
                            />).JSON;
    
    expect(compareSnapshot).toMatchSnapshot();
  })

})
