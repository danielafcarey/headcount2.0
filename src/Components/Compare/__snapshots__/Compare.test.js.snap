// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Compare matches the snapshot when one card is in compareCards prop 1`] = `undefined`;

exports[`Compare matches the snapshot when two cards in compareCards prop 1`] = `undefined`;

exports[`Compare matches the snapshot with no cards 1`] = `undefined`;
